src/admin/content-block/helpers/generators/media-grid.ts

```diff
@@ -0,0 +1,64 @@
+import i18n from '../../../../shared/translations/i18n';
+
+import {
+	ContentBlockConfig,
+	ContentBlockEditor,
+	ContentBlockType,
+	MediaGridBlockComponentState,
+	MediaGridBlockState,
+} from '../../../shared/types';
+import { BLOCK_FIELD_DEFAULTS, BLOCK_STATE_DEFAULTS, TEXT_FIELD } from './defaults';
+
+export const INITIAL_MEDIA_GRID_COMPONENTS_STATE = (): MediaGridBlockComponentState[] => [{}];
+
+export const INITIAL_MEDIA_GRID_BLOCK_STATE = (position: number): MediaGridBlockState => ({
+	...BLOCK_STATE_DEFAULTS(ContentBlockType.MediaGrid, position),
+	ctaTitle: '',
+	ctaContent: '',
+	ctaButtonLabel: '',
+	ctaButtonAction: { type: 'ITEM', value: '' },
+});
+
+export const MEDIA_GRID_BLOCK_CONFIG = (position: number = 0): ContentBlockConfig => ({
+	name: i18n.t('Media grid'),
+	type: ContentBlockType.MediaGrid,
+	components: {
+		name: i18n.t('Media item'),
+		limits: {
+			min: 1,
+			max: 8,
```

Comment:
moet er een max op staan? Ik dacht dat ze die zelf moesten ingeven?

Reply:
zelf ingeven gaat nog toegevoegd worden eens de zoek query mogelijk is in de content-picker, ik zal mss wel gwn die max er af laten dan.

Reply:
Fixed